### PR TITLE
WIP: Add `stderr` support

### LINKED ISSUE #4: Stderr support
[Issuehunt badges](https://issuehunt.io/r/sindresorhus/terminal-link/issues/4)

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Backers (Total: $40.00)

- [issuehunt issuehunt](https://issuehunt.io/u/issuehunt) ($40.00)
### Submitted pull Requests
- [#5 Stderr support](https://issuehunt.io/r/sindresorhus/terminal-link/pull/5)
---

### Tips

- Checkout the [Issuehunt explorer](https://issuehunt.io/r/sindresorhus/terminal-link/) to discover more funded issues.
- Need some help from other developers? [Add your repositories](https://issuehunt.io/r/new) on IssueHunt to raise funds.
---
IssueHunt has been backed by the following sponsors. [Become a sponsor](https://issuehunt.io/membership/members)
</details>

### PR DESCRIPTION
**Pull request by tom-sherman**
Closes #4 

This PR allows you to do the following:

```javascript
// stdout
terminalLink('Google', 'https://google.com');
terminalLink.isSupported;

// stderr
terminalLink.stderr('Google', 'https://google.com');
terminalLink.stderr.isSupported;
```

### CODE REVIEW

**Comment by tom-sherman** [comment]
The tests are failing because of the typings that I've amended. Would be good if someone with more Typescript knowledge than I could help 😄

**Comment by sindresorhus** [comment]
@tom-sherman You can see how it's handled here: https://github.com/sindresorhus/write-json-file/blob/master/index.d.ts

File: readme.md [hunk 1]
```diff
@@ -30,6 +30,8 @@ console.log(link);
 
 For unsupported terminals, the link will be printed in parens after the text: `My website (https://sindresorhus.com)`.
 
+Use `terminalLink.stderr(text, url, [options])` for outputting to stderr.
```
Comment (sindresorhus) [inline readme.md]: This makes it sound like the function call outputs to stderr, which it doesn't.

Comment (tom-sherman) [reply readme.md]: How about "Use [...] to create a link for use in stderr."?

Comment (sindresorhus) [reply readme.md]: 👍

**Comment by tom-sherman** [comment]
@sindresorhus All completed :-)

**Comment by tom-sherman** [comment]
Not sure what's going on, but I can't get the tsd tests to pass after updating from master.

```
  index.d.ts:18:67
  ×  18:67  Cannot find name Options.

  index.test-d.ts:6:19
  ×   6:19  Cannot invoke an expression whose type lacks a call signature. Type typeof import("D:/terminal-link/index") has no compatible call signatures.
  ×   9:1   Cannot invoke an expression whose type lacks a call signature. Type typeof import("D:/terminal-link/index") has no compatible call signatures.
  ×  10:13  Parameter text implicitly has an any type.
  ×  10:19  Parameter url implicitly has an any type.
  ×  14:20  Cannot find name isSupported.
  ×  18:32  Property stderr does not exist on type typeof import("D:/terminal-link/index").
  ×  20:32  Property stderr does not exist on type typeof import("D:/terminal-link/index").
  ×  21:12  Parameter text implicitly has an any type.
  ×  21:18  Parameter url implicitly has an any type.
  ×  24:33  Property stderr does not exist on type typeof import("D:/terminal-link/index").

  11 errors
```

These errors also appear in VSCode, could it be the version of Typescript I have globally installed?

```
PS D:\terminal-link> tsc --version
message TS6029: Version 1.5.3
```

Any help here is appreciated

**Comment by sindresorhus** [comment]
You didn't correctly merge in `master`. Those errors are TS type errors. You need to resolve them.

For example:

> ×  14:20  Cannot find name isSupported.

is obvious if you look at your diff.

**Pull request closed without merging**